### PR TITLE
front end

### PR DESCRIPTION
**Pull request by alexander871015**

### CODE REVIEW

File: Hologram-Data-Availability/hda-web-server/main/hda/hda.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+void HDAHandler::start() {
+    server.serve();
+}
+
+void HDAHandler::getLastRefreshed(const Rest::Request& request, 
+    Http::ResponseWriter response) {
+    UNUSED(request);
+    response.send(Http::Code::Ok, "Wed May 19 15:46:11 2020");
```
Comment (sarahchen-google) [inline Hologram-Data-Availability/hda-web-server/main/hda/hda.cpp:53]: Add TODO here saying "will apply actual value here. Currently it is for prototyping purposes".

File: Hologram-Data-Availability/hda-web-server/WORKSPACE [hunk 2]
```diff
@@ -0,0 +1,65 @@
+workspace(name = "cpp_server")
```
Comment (sarahchen-google) [inline Hologram-Data-Availability/hda-web-server/WORKSPACE:1]: Change the folder structure into

/hologram_tools
  /server
  /front_end

Comment (sarahchen-google) [reply Hologram-Data-Availability/hda-web-server/WORKSPACE:1]: /hologram_tools
->/server
->/front_end

File: Hologram-Data-Availability/hda-web-server/main/hda/hda.h [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        HDAHandler(Address addr);
+
+        void start();
+    private:
+        Rest::Router router;
+        Http::Endpoint server;
+
+        void getDashboard(const Rest::Request& request, Http::ResponseWriter response);
+
+        void sendCommand(const Rest::Request& request, Http::ResponseWriter response);
```
Comment (emilyqu) [inline Hologram-Data-Availability/hda-web-server/main/hda/hda.h:35]: nit: add comment on each function to explain what it does.

File: Hologram-Data-Availability/hda-web-server/WORKSPACE (hunk 2 above)
Comment (sarahchen-google) [reply Hologram-Data-Availability/hda-web-server/WORKSPACE:1]: /hologram_data_availability
BUILD
->/server
   -> server_main.cc
->/front_end

**Pull request closed without merging**